hazelcast/src/main/java/com/hazelcast/instance/AbstractMember.java

```diff
@@ -43,7 +42,7 @@
 import static com.hazelcast.util.Preconditions.checkNotNull;
 
 @PrivateApi
-public abstract class AbstractMember implements Member, Versioned {
+public abstract class AbstractMember implements Member {
```

Comment:
`AbstractMember` has also `V3_12` version checks in serialization methods.

Reply:
Wow, we have even a test for it... surprised!
```
java.lang.AssertionError: Expected com.hazelcast.client.impl.MemberImpl to implement Versioned, since in.getVersion() is used
```

Reply:
Yup. It's best effort IIRC as it may have false-negatives in case the serialisation code fails with an exception before even getting to the version check (e.g. by throwing a NPE). But it's really useful!

Reply:
Fixed, thanks!